### PR TITLE
[#173432184] - notification messages adjustments

### PR DESCRIPTION
**Pull request by AleDore**
Updated content messages. Now some message types accept a Date that fills the dynamic part of text.

### CODE REVIEW

File: EligibilityCheckOrchestrator/index.ts [hunk 1]
```diff
@@ -37,6 +38,27 @@ export type OrchestratorInput = t.TypeOf<typeof OrchestratorInput>;
...
+    case "EligibilityCheckSuccessEligibleWithDiscrepancies":
+      return MESSAGES[messageType](validBefore);
+    case "EligibilityCheckSuccessIneligible":
+    case "EligibilityCheckFailure":
+    case "EligibilityCheckConflict":
+    case "BonusActivationSuccess":
+    case "BonusActivationFailure":
+      return MESSAGES[messageType]();
+    default:
+      throw new Error(
```
Comment (balanza) [inline EligibilityCheckOrchestrator/index.ts:56]: I suggest the use of `assertNever` so we can be warned at compile time for a missing case.

File: EligibilityCheckOrchestrator/index.ts [hunk 2]
```diff
@@ -37,6 +38,27 @@ export type OrchestratorInput = t.TypeOf<typeof OrchestratorInput>;
...
+        `Cannot get MessageContent on EligibilityCheckOrchestrator`
+      );
+  }
+};
+
```
Comment (balanza) [inline EligibilityCheckOrchestrator/index.ts:61]: I'd rather export this function from `utils/messages` as a helper

File: EligibilityCheckOrchestrator/index.ts (hunk 1 above)
Comment (AleDore) [reply EligibilityCheckOrchestrator/index.ts]: done

File: EligibilityCheckOrchestrator/index.ts (hunk 2 above)
Comment (AleDore) [reply EligibilityCheckOrchestrator/index.ts]: done

**Pull request merged**